system/kube-monitoring/charts/prometheus-collector/templates/_prometheus.yaml.tpl

```diff
@@ -312,6 +312,29 @@ scrape_configs:
     replacement: {{ $region }}
 {{ end }}
 
+{{ range $region := .Values.global.regions }}
+- job_name: 'blackbox-tcp-{{ $region }}'
+  metrics_path: /probe
+  scrape_interval: 15s
+  params:
+    module: [tcp_connect]
+  static_configs:
+    - targets:
+      {{- range $.Values.blackbox_exporter.tcp_probe_targets }}
+      - {{ . }}
+      {{- end }}
+  scheme: https
+  relabel_configs:
+  - source_labels: [__address__]
+    target_label: __param_target
+  - source_labels: [__param_target]
+    target_label: instance
+  - target_label: __address__
+    replacement: prober.{{ $region }}.cloud.sap
+  - target_label: region_probed_from
+    replacement: {{ $region }}
+{{ end }}
+
```

Comment:
ok for now, but for the next iteration: move to separate `extra_scrape_config`, just as we do in [kubernikus](https://github.com/sapcc/kubernikus/blob/master/charts/kubernikus-system/charts/prometheus/templates/config.yaml#L21-L23)

Reply:
Of course. I do not wish for this to stay there.